public/index.html

```diff
@@ -31,7 +31,12 @@
   </head>
   <body>
     <noscript>You need to enable JavaScript to run this app.</noscript>
-    <div id="root"></div>
+    <div id="root">
+      <div class="splash">
+        <div class = "splashy">
+        <img src="./logo512.png" />
+      </div>
+    </div>
```

Comment:
missing a closing `</div>` here. also you could move this to `App.js` here

https://github.com/Hack-Weekly/fuchsia-cuttlefish-chat-app/blob/a6dfc0ac2e9d0d193f9b8d9b9ae500589ae37ef6/src/App.js#L11-L12

and you could move everything in Splash.js here too, and maybe use `useEffect` and `useRef` instead? here's an example of how that would work:

https://github.com/Hack-Weekly/fuchsia-cuttlefish-chat-app/blob/a6dfc0ac2e9d0d193f9b8d9b9ae500589ae37ef6/src/components/SendMessage.js#L12-L14